plotter/labels.go

```diff
@@ -96,11 +96,16 @@ func (l *Labels) DataRange() (xmin, xmax, ymin, ymax float64) {
 // plot.GlyphBoxer interface.
 func (l *Labels) GlyphBoxes(p *plot.Plot) []plot.GlyphBox {
 	bs := make([]plot.GlyphBox, len(l.Labels))
+	offset := vg.Point{X: l.XOffset, Y: l.YOffset}
 	for i, label := range l.Labels {
-		bs[i].X = p.X.Norm(l.XYs[i].X)
-		bs[i].Y = p.Y.Norm(l.XYs[i].Y)
-		sty := l.TextStyle[i]
-		bs[i].Rectangle = sty.Rectangle(label)
+		rect := l.TextStyle[i].Rectangle(label)
```

Comment:
Do we want to add a `Rectangle.Add(vg.Point)`? There is precedent in image.

Reply:
sure.
I'll try to insert this before this commit and rebase.
propagating to image will probably another PR.

Reply:
Sorry, I meant that stdlib's `image.Rectangle` has an `Add` method.